### PR TITLE
bug fix e6

### PR DESCRIPTION
**Pull request by bradymiller**
bug fix

### CODE REVIEW

File: portal/report/portal_custom_report.php [hunk 1]
```diff
@@ -600,7 +600,7 @@ function next_prev(action){
 while ($result = sqlFetchArray($inclookupres)) {
   // include_once("{$GLOBALS['incdir']}/forms/" . $result["formdir"] . "/report.php");
     $formdir = $result['formdir'];
-    if (substr($formdir, 0, 3) == 'LBF') {
+    if (substr($formdir, 0, 3) == 'LBF' || substr($formdir, 0, 3) == 'HIS') {
```
Comment (bradymiller) [inline portal/report/portal_custom_report.php:603]: going to revert above line

File: portal/report/portal_custom_report.php [hunk 2]
```diff
@@ -1006,7 +1006,7 @@ function next_prev(action){
                 ?>
                 <div name="search_div" id="search_div_<?php echo attr($form_id)?>_<?php echo attr($res[1])?>" class="report_search_div class_<?php echo attr($res[1]); ?>">
                 <?php
-                if (substr($res[1], 0, 3) == 'LBF') {
+                if (substr($res[1], 0, 3) == 'LBF' || substr($formdir, 0, 3) == 'HIS') {
```
Comment (bradymiller) [inline portal/report/portal_custom_report.php:1009]: going to revert above line

**Comment by bradymiller** [comment]
testing well, so bringing this in

**Comment by stephenwaite** [comment]
nice

**Pull request merged**